files/dashboards/k8s-resources-node.json

```diff
@@ -619,7 +671,7 @@
                      "step": 10
                   },
                   {
-                     "expr": "sum(kube_pod_container_resource_requests{cluster_id=\"$cluster\", node=~\"$node\", resource=\"memory\"}) by (pod)",
+                     "expr": "sum(cluster:namespace:pod_memory:active:kube_pod_container_resource_requests{cluster_id=\"$cluster\", node=~\"$node\"}) by (pod)",
```

Comment:
Will this work? Where is this recording rule coming from?

Reply:
From here https://monitoring.mixins.dev/kubernetes/

Reply:
I'm not sure we have them

Reply:
They are included in kube-prometheus-stack though

Reply:
We can just not update them :)